charts/camunda-platform-alpha/test/unit/web-modeler/configmap_restapi_test.go

```diff
@@ -22,6 +23,13 @@ type configmapRestAPITemplateTest struct {
 	templates []string
 }
 
+var requiredValues = map[string]string{
+	"webModeler.enabled":                                  "true",
+	"webModeler.restapi.mail.fromAddress":                 "example@example.com",
+	"global.identity.auth.connectors.existingSecret.name": "foo",
+	"global.identity.auth.core.existingSecret.name":       "foo",
```

Comment:
These values are required now given that `global.testDeprecationFlags.existingSecretsMustBeSet` was changed from `warning` to `error`. I think they're missing in all of the unit tests. At least, I added them here to be able to run the (new) tests for the restapi.